pkg/minikube/cni/bridge.go

```diff
@@ -84,6 +85,10 @@ func (c Bridge) Apply(r Runner) error {
 		return fmt.Errorf("bridge CNI is incompatible with multi-node clusters")
 	}
 
+	if _, err := r.RunCmd(exec.Command("sudo", "mkdir", "-p", "/etc/cni/net.d")); err != nil {
+		return errors.Wrap(err, "netconf")
```

Comment:
some error message about `sudo mkdir`, not `netconf` here, may be

Reply:
Just returning the error instead of wrapping would probably work too